From d8dddb115d296d8980cebee2527d6b4b875265b5 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Fri, 15 Jul 2011 12:38:28 -0400
Subject: sound: Add module.h to the previously silent sound users

Lots of sound drivers were getting module.h via the implicit presence
of it in <linux/device.h> but we are going to clean that up.  So
fix up those users now.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
(cherry picked from commit da155d5b40587815a4397e1a69382fe2366d940b)

Cherry-picked changes to:
	sound/soc/sh/fsi-ak4642.c
	sound/soc/sh/fsi-da7210.c
	sound/soc/sh/fsi.c

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 sound/soc/sh/fsi-ak4642.c | 1 +
 sound/soc/sh/fsi-da7210.c | 1 +
 sound/soc/sh/fsi.c        | 1 +
 3 files changed, 3 insertions(+)

diff --git a/sound/soc/sh/fsi-ak4642.c b/sound/soc/sh/fsi-ak4642.c
index 770a71a..dff64b9 100644
--- a/sound/soc/sh/fsi-ak4642.c
+++ b/sound/soc/sh/fsi-ak4642.c
@@ -10,6 +10,7 @@
  */
 
 #include <linux/platform_device.h>
+#include <linux/module.h>
 #include <sound/sh_fsi.h>
 
 struct fsi_ak4642_data {
diff --git a/sound/soc/sh/fsi-da7210.c b/sound/soc/sh/fsi-da7210.c
index 59553fd..f5586b5b 100644
--- a/sound/soc/sh/fsi-da7210.c
+++ b/sound/soc/sh/fsi-da7210.c
@@ -11,6 +11,7 @@
  */
 
 #include <linux/platform_device.h>
+#include <linux/module.h>
 #include <sound/sh_fsi.h>
 
 static int fsi_da7210_init(struct snd_soc_pcm_runtime *rtd)
diff --git a/sound/soc/sh/fsi.c b/sound/soc/sh/fsi.c
index a32fd16..3d7016e 100644
--- a/sound/soc/sh/fsi.c
+++ b/sound/soc/sh/fsi.c
@@ -16,6 +16,7 @@
 #include <linux/pm_runtime.h>
 #include <linux/io.h>
 #include <linux/slab.h>
+#include <linux/module.h>
 #include <sound/soc.h>
 #include <sound/sh_fsi.h>
 
-- 
1.7.10.2.565.gbd578b5

